### ISSUE TITLE
ESNext

### ISSUE THREAD

**Issue by eskinner31** [AtomLinter/linter-jscs#245]
### Version
4.0.0

### Issue
Every time I open a JS file I'm getting this warning

```
Error: Config values to remove in 3.0: The esnext option is enabled by default.
```

I've tried removing the config values but that did not help. Full error message below.

```
Error: Config values to remove in 3.0:
The `esnext` option is enabled by default.
    at NodeConfiguration.Configuration._errorOnRemovedOptions (/Users/Skinner/.atom/packages/linter-jscs/node_modules/jscs/lib/config/configuration.js:401:15)
    at NodeConfiguration.Configuration._processConfig (/Users/Skinner/.atom/packages/linter-jscs/node_modules/jscs/lib/config/configuration.js:420:10)
    at NodeConfiguration.Configuration.load (/Users/Skinner/.atom/packages/linter-jscs/node_modules/jscs/lib/config/configuration.js:203:10)
    at [object Object].StringChecker.configure (/Users/Skinner/.atom/packages/linter-jscs/node_modules/jscs/lib/string-checker.js:62:29)
    at [object Object].Checker.configure (/Users/Skinner/.atom/packages/linter-jscs/node_modules/jscs/lib/checker.js:28:39)
    at Object.lint (/Users/Skinner/.atom/packages/linter-jscs/src/linter-jscs.js:145:19)
    at promises.push.Promise.then._this.emitter.emit.linter (/Users/Skinner/.atom/packages/linter/lib/linter-registry.js:63:26)
    at /Users/Skinner/.atom/packages/linter/lib/linter-registry.js:62:23
    at Set.forEach (native)
    at LinterRegistry.lint (/Users/Skinner/.atom/packages/linter/lib/linter-registry.js:60:18)
    at /Users/Skinner/.atom/packages/linter/lib/linter.coffee:98:16
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at /Users/Skinner/.atom/packages/linter/lib/editor-linter.js:80:22
    at /Users/Skinner/.atom/packages/linter/lib/helpers.coffee:23:18
```

**Comment by Arcanemagus** [comment]
Please read the error message and remove the value in your configuration file that it is referencing. `jscs` v3.0.0 removed several configuration settings that no longer make sense for it.
